.circleci/d9-readiness.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+IFS=$'\n\t'
+set -euo pipefail
+
+LOG=/app/d9-readiness.log
+
+composer require mglaman/drupal-check
+rm -f web/profiles/contrib/govcms/composer.json
+rm -f "${LOG}" && touch "${LOG}"
+
+set +e
+echo echo -e "\nRUNNING DEPRECATION TEST FOR: GovCMS Profile"
+vendor/bin/drupal-check --no-progress web/profiles/contrib/govcms > "${LOG}"
+set -e
+
+for module in web/modules/contrib/*; do
+    if [ -d "${module}" ]; then
+        echo -e "\nRUNNING DEPRECATION TEST FOR: ${module}"
```

Comment:
can we echo this into the LOG as well as the screen - the context for the line errors may be hard to interpret otherwise :-)

Reply:
Yep, it was meant to go to the log, fixing that now.